Tmap8 clean up tmap7 (#295)
idaholab/TMAP8: doc/content/verification_and_validation/val-2d.md

C = 0.
\end{equation}

> Due to the high thermal conductivity in Tungsten, the model simplifies the thermal diffusion as a instantaneous process to increase the model efficiency. The temperature inside the Tungsten sample is consistent with the temperature on the surface of Tungsten sample. The temperatures is 300 K during implantation, and the temperature after implantation on the upstream and downstream sides are defined as:

Reviewer: ```suggestion
Due to the high thermal conductivity in Tungsten, the model simplifies the thermal diffusion as an instantaneous process to increase the model efficiency. That is, the temperature inside the Tungsten sample is consistent with the temperature on the surface of Tungsten sample. The temperature is set to 300 K during implantation, and the temperature after implantation on the upstream and downstream sides are defined as:
```
